iterating-over-data/practice-iterating-over-data.md

```diff
@@ -0,0 +1,465 @@
+# Practice: Iterating Over Data
+
+## Directions
+
+Complete all questions below.
+
+## Practice
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: EJew-RW0t
+* title: Which Range is Correct?
+
+##### !question
+
+Given a function called `stopwatch` that takes in the argument `22`, which of the following loops would  correctly start at `1` and increment all the way up to *and include* `22`?
+
+##### !end-question
+
+##### !options
+
+* `for i in 22:`
+* `for i in range(22)`
+* `for i in range(1,22)`
+* `for i in range(1,23)`
```

Comment:
```suggestion
* `for i in range(22):`
* `for i in range(1,22):`
* `for i in range(1,23):`
```

Adding in consistency in syntax

Reply:
Nitpick: Is there a reason why we aren't doing a space between `1,` and `2X`?

```suggestion
* `for i in range(22):`
* `for i in range(1, 22):`
* `for i in range(1, 23):`
```